src/app/zap-templates/zcl/data-model/chip/ethernet-network-diagnostics-cluster.xml

```diff
@@ -35,14 +35,14 @@ limitations under the License.
     <code>0x0037</code>
     <define>ETHERNET_NETWORK_DIAGNOSTICS_CLUSTER</define>
     <description>The Ethernet Network Diagnostics Cluster provides a means to acquire standardized diagnostics metrics that MAY be used by a Node to assist a user or Administrative Node in diagnosing potential problems.</description>
-    <attribute side="server" code="0x00" define="PHY_RATE" type="ENUM8" writable="false" optional="true">PHYRate</attribute>
-    <attribute side="server" code="0x01" define="FULL_DUPLEX" type="BOOLEAN" min="0x00" max="0x01" writable="false" default="0x00" optional="true">FullDuplex</attribute>
+    <attribute side="server" code="0x00" define="PHY_RATE" type="ENUM8" writable="false" isNullable="true" optional="true">PHYRate</attribute>
+    <attribute side="server" code="0x01" define="FULL_DUPLEX" type="BOOLEAN" min="0x00" max="0x01" writable="false" isNullable="true" optional="true">FullDuplex</attribute>
     <attribute side="server" code="0x02" define="PACKET_RX_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">PacketRxCount</attribute>
     <attribute side="server" code="0x03" define="PACKET_TX_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">PacketTxCount</attribute>
     <attribute side="server" code="0x04" define="TX_ERR_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">TxErrCount</attribute>
     <attribute side="server" code="0x05" define="COLLISION_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">CollisionCount</attribute>
     <attribute side="server" code="0x06" define="ETHERNET_OVERRUN_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">OverrunCount</attribute>
```

Comment:
These should all be optional, since they are only present if certain features are supported, no?

Reply:
The spec has defined non null default value for those attributes, and does not say "A value of null SHALL indicate that the interface is not currently configured or operational." for those attributes. 

If related feature is not available, I think it make sense to me just return 0 for those counts

Reply:
The point is, per spec the attributes are allowed to not be present at all, as long as the feature flag is not set.  This XML is requiring them to always be present, which adds additional space/memory requirements that the spec does not mean to impose.

Please just implement the spec here instead of making up something else that "makes sense".

Filed https://github.com/project-chip/connectedhomeip/issues/14932